<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-55TGAJ2" failures="1" tests="2" name="Accounts Test on chrome" time="35.891" errors="0" timestamp="24 May 2021 13:14:24 GMT">
  <testcase classname="testcases.VerifyAccountsModule" name="@AfterMethod CloseDriver" time="0.0">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [https://abccom359-dev-ed.my.salesforce.com/setup/forcecomHomepage.apexp?setupid=ForceCom] but found [https://abccom359-dev-ed--c.documentforce.com/secur/contentDoor?startURL=https%3A%2F%2Fabccom359-dev-ed.my.salesforce.com%2Fsetup%2FforcecomHomepage.apexp%3Fsetupid%3DForceCom&amp;amp;sid=00D5e000000J7Y4%21AQkAQDBgXXsmukRAvsqDy_ARHCwuvnnK82sgmDifY024M2Ca5XlGdJaZvWrKGcTUGC4.wdpdII00DpWceXtd2urjM.Dy3Jjd&amp;amp;skipRedirect=1&amp;amp;lm=eyJlbmMiOiJBMjU2R0NNIiwiYXVkIjoiMDBENWUwMDAwMDBKN1k0Iiwia2lkIjoie1widFwiOlwiMDBENWUwMDAwMDBKN1k0XCIsXCJ2XCI6XCIwMkc1ZTAwMDAwMFBuU2tcIixcImFcIjpcImNvbnRlbnRkb29ydXNlcnRyYW5zaWVudGtleWVuY3J5cHRcIixcInVcIjpcIjAwNTVlMDAwMDAwcVhzbVwifSIsImNyaXQiOlsiaWF0Il0sImlhdCI6MTYyMTg2MjA1MjcyMSwiZXhwIjowfQ%3D%3D..iqojOJ7grTZPXflx.pagRrD-6Q0k7EFB2wtNpJw%3D%3D.Iqa_MMWmrLgl2iBFi3Ioyg%3D%3D]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [https://abccom359-dev-ed.my.salesforce.com/setup/forcecomHomepage.apexp?setupid=ForceCom] but found [https://abccom359-dev-ed--c.documentforce.com/secur/contentDoor?startURL=https%3A%2F%2Fabccom359-dev-ed.my.salesforce.com%2Fsetup%2FforcecomHomepage.apexp%3Fsetupid%3DForceCom&sid=00D5e000000J7Y4%21AQkAQDBgXXsmukRAvsqDy_ARHCwuvnnK82sgmDifY024M2Ca5XlGdJaZvWrKGcTUGC4.wdpdII00DpWceXtd2urjM.Dy3Jjd&skipRedirect=1&lm=eyJlbmMiOiJBMjU2R0NNIiwiYXVkIjoiMDBENWUwMDAwMDBKN1k0Iiwia2lkIjoie1widFwiOlwiMDBENWUwMDAwMDBKN1k0XCIsXCJ2XCI6XCIwMkc1ZTAwMDAwMFBuU2tcIixcImFcIjpcImNvbnRlbnRkb29ydXNlcnRyYW5zaWVudGtleWVuY3J5cHRcIixcInVcIjpcIjAwNTVlMDAwMDAwcVhzbVwifSIsImNyaXQiOlsiaWF0Il0sImlhdCI6MTYyMTg2MjA1MjcyMSwiZXhwIjowfQ%3D%3D..iqojOJ7grTZPXflx.pagRrD-6Q0k7EFB2wtNpJw%3D%3D.Iqa_MMWmrLgl2iBFi3Ioyg%3D%3D]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.VerifyAccountsModule.CloseDriver(VerifyAccountsModule.java:40)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:712)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod CloseDriver -->
  <testcase classname="testcases.VerifyAccountsModule" name="TC10_CreateAccount" time="14.173"/>
  <testcase classname="testcases.VerifyAccountsModule" name="TC12_Editview" time="14.053">
    <failure type="org.openqa.selenium.NoSuchElementException" message="Cannot locate element with text: NirmalaShanmugamani
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;DESKTOP-55TGAJ2&amp;apos;, ip: &amp;apos;192.168.0.25&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;15.0.1&amp;apos;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Cannot locate element with text: NirmalaShanmugamani
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-55TGAJ2', ip: '192.168.0.25', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '15.0.1'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.Select.selectByVisibleText(Select.java:147)
	at base.BaseTest.SelectByText(BaseTest.java:171)
	at testcases.VerifyAccountsModule.TC12_Editview(VerifyAccountsModule.java:113)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC12_Editview -->
  <testcase classname="testcases.VerifyAccountsModule" name="TC11_CreateNewView" time="0.0">
    <ignored/>
  </testcase> <!-- TC11_CreateNewView -->
</testsuite> <!-- Accounts Test on chrome -->
